src/src/subjectslist/subjectslist.controller.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  Controller,
+  Get,
+  Post,
+  Body,
+  Patch,
+  Param,
+  Delete,
+} from '@nestjs/common';
+import { SubjectslistService } from './subjectslist.service';
+import { CreateSubjectslistDto } from './dto/create-subjectslist.dto';
+import { UpdateSubjectslistDto } from './dto/update-subjectslist.dto';
+
+@Controller('subjectslist')
+export class SubjectslistController {
+  constructor(private readonly subjectslistService: SubjectslistService) {}
+
+  @Post('/create')
```

Comment:
make all routes protected for - add /edit / delete

or comment all excecpt the for getting list of subjects